### DISCUSSION TITLE
Premissions for internal Groups

### DISCUSSION THREAD

**Discussion by simpsonetti** [Ylianst/MeshCentral#5784] [Q&A]
Hi, i'm trying to set permissions for the "regular" users who logs in. For example only Administrator should add Device Groups, new Groups and so on. So i wan't to setup these permissions as default. 
Where do i do this?

**Comment by si458** [comment]
Add the user, then select the user, then underneath specify what groups they have access too and what permissions they have

You can also click the pencil next to Features and Server Rights, to specify what admin permissions that user has

Edit. You can also create a user group with your device permissions, then add ur users to the group and they inherit the permissions u set

**Reply by simpsonetti** [reply]
Yes, this is the way i already know. But, if there are no other Groups until now, i want to setup the default permissions for EVERY user to not add groups, device groups ... 
I don't see an other option/way so setup this. 
Or is there a config option?
